safe_control_gym/controllers/mpc/mpc_utils.py

```diff
@@ -90,3 +92,59 @@ def reset_constraints(constraints):
     if len(constraints_list.input_state_constraints) > 0:
         raise NotImplementedError('[Error] Cannot handle combined state input constraints yet.')
     return constraints_list, state_constraints_sym, input_constraints_sym
+
+
+def set_acados_constraint_bound(constraint,
+                                bound_type,
+                                bound_value=None,
+                                ):
+    '''Set the acados constraint bound.
```

Comment:
ideally add full args and returns for new functions. Its kind of a pain but will improve documentation, especially later if we decide to use a documentation engine like sphinx

Reply:
I added full args to the one you pointed out but not sure whether there is still some missing one / old ones without full args. Might will address this more carefully in the future PR.